lib/rex/proto/http/client.rb

```diff
@@ -731,6 +764,13 @@ def peerinfo
   #
   attr_accessor :ntlm_client
 
+  def get_file_data(file)
+    file.respond_to?('read') ? (contents = file.read; file.rewind; contents) : file.to_s
```

Comment:
Normally we wouldn't expect an API to automatically rewind a file/stream, I think we can remove that logic :+1: